README.md

```diff
@@ -100,19 +82,16 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **[Relational Database based on real-life]**
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+> coming soon...
```

Comment:
- [ ] Good job updating the readme file as suggested by the previous reviewer, but please consider removing the `Live demo` section from the readme file. It is only required when you have a project that is deployed. This is a mandatory requirement for all projects.